src/InvertMouse/MainForm.Designer.cs

```diff
@@ -220,9 +220,14 @@ private void InitializeComponent()
             // 
             // MainForm
             // 
+            int minWidth = 311;
```

Comment:
This file should not be edited manually. It is generated by Windows Form Designer.

Reply:
You're right, it's not generally recommended to alter this file. Doing this puts a burden on the maintainer to restore such edits if the file is regenerated. I tried to set the values outside of this file, but that broke the scaling.